.github/workflows/ci.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm test

```

Comment:
Why did you remove npm test ?

Reply:
changed it out for npm publish --dry-run, which also includes test

Reply:
It's broken if npm publish runs the tests. How come ?